dockci/models/job.py

```diff
@@ -57,5 +57,7 @@ def _all_builds(self):
     slug = None
     repo = LoadOnAccess(default=lambda _: '')
     name = LoadOnAccess(default=lambda _: '')
+    hipchat_api_token = LoadOnAccess(default=lambda _: '')
```

Comment:
hmm in the future, we should encrypt this so that personal API tokens aren't public in the git repo. can you add a # TODO for when we add users please?